### ISSUE TITLE
Get property value query editor settings

### ISSUE THREAD

**Issue by hwandersman** [grafana/iot-sitewise-datasource#328]
**What happened**:
Get property value query displays settings for the quality, time ordering, and format, when none are used.

**What you expected to happen**:
Should not display these settings since there is no use for these in the API: https://docs.aws.amazon.com/iot-sitewise/latest/APIReference/API_GetAssetPropertyValue.html

This will lead to a confusing experience for customers.

**How to reproduce it (as minimally and precisely as possible)**:
1. Add a Get property value query in the query editor
2. Try to set the quality, time ordering, or format and nothing changes since it is not used

**Screenshots**

<img width="1102" alt="image" src="https://github.com/grafana/iot-sitewise-datasource/assets/87385528/14140c44-b605-40db-aec9-30dcb4893bf5">

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.4.0
- Plugin version: 1.20.0
- OS Grafana is installed on: MacOS
- User OS & Browser: Sonoma 14.5, Chrome
- Others: Amazon Managed Grafana environment

**Comment by sarahzinger** [comment]
Makes sense to me! Let's hide those settings, shouldn't be too difficult. Sharing with the team to prioritize a fix.
